src/App.res

```diff
@@ -2,7 +2,7 @@
 let make = () => {
   let (pokemonList: option<PokemonList.t>, setPokemonList) = React.useState(() => None)
 
-  React.useEffect(() => {
+  React.useEffect0(() => {
```

Comment:
You can use an `useEffect1` with `setPokemonList` in the dependencies

Reply:
Yes, seems more accurate, thanks :)